### ISSUE TITLE
Remove "Fast"-suffix from moves

### ISSUE THREAD

**Issue by BrunnerLivio** [BrunnerLivio/pokemongo-json-pokedex#30]
Some moves have "Fast" in their name. In pokemongo though, this suffix is not present. We should remove that too.

**Comment by vfcp** [comment]
I'm also looking for a elegant solution for this.

The in-game move name comes from localization strings, not from game master file. I don't see this as a bug as propose of this project is to translate game master file to json format.

Similarly to game, maybe we should use separate files for localization in our projects (moves and pokemon)? Something like in PokeAlarm code: https://github.com/PokeAlarm/PokeAlarm/tree/master/locales

**Comment by BrunnerLivio** [comment]
@vfcp 

Good point. 

But what I don't want to maintain is the localization strings by myself. 

The Pokémon / Move strings could probably be fetched from PokéAPI or so. But I don't know if it is easily possible to fetch the PokemonGO-only localisation strings (e.g. Avatar Customization stuff). 

 And it is not the purpose of this repository to have localization files in it. This repository should only fetch its information from other reliable sources (pokemongo-game-master / PokéAPI) and not store them by itself.

**Comment by vfcp** [comment]
I agree, the localization strings is not on scope of this repo. This tool propose is to have an accurate representation of game master file and localization is not part of it. My previous message was not clear, but when proposing localization files, I meant in our projects repos, if needed. This is why I don't see this issue as a bug.

The removal of "Fast" -suffix is not the only issue. For example "Mud Slap Fast" is "Mud-Slap" in game. There might be other slight differences and Niantics can change move names in any update. It would be a pain to maintain this manually and I need to research to see how to extract this from game resources.

Personally, I'll use localization as it fits my project long term plans. When I find a good source, I'll share it in this thread for those that might be looking for something similar.
